src/ducks/modules/stage.js

```diff
@@ -1,13 +1,18 @@
+import { actionTypes as protocolActionTypes } from '../../ducks/modules/protocol';
+
 const NEXT_PROMPT = 'NEXT_PROMPT';
 const PREVIOUS_PROMPT = 'PREVIOUS_PROMPT';
+const NEXT_STAGE = 'NEXT_STAGE';
+const PREVIOUS_STAGE = 'PREVIOUS_STAGE';
 
 const initialState = {
   promptIndex: 0,
```

Comment:
Okay, I've tracked down why this is in the state: The interface needs to know which prompt is active when creating nodes so it can append the appropriate attributes.

It's great that you brought this up though, it's made me reconsider how the current stage is propagated though nested components. They probably should get it as a prop from the interface and not direct from the state.

Still think it the module should have a better name though.